components/script/script_thread.rs

```diff
@@ -1271,7 +1271,7 @@ impl ScriptThread {
                 match fetch_data {
                     FetchResponseMsg::ProcessResponse(metadata) => self.handle_fetch_metadata(id, metadata),
                     FetchResponseMsg::ProcessResponseChunk(chunk) => self.handle_fetch_chunk(id, chunk),
-                    FetchResponseMsg::ProcessResponseEOF(eof) => self.handle_fetch_eof(id, eof),
+                    FetchResponseMsg::ProcessResponseEOF(eof) => self.handle_fetch_eof(id, eof.map(|_| ())),
```

Comment:
This resource timing has a bunch of useful information in it, including redirect counts and fetch start (presumably). Throwing it away is probably not the right choice, since it represents the document navigation response.